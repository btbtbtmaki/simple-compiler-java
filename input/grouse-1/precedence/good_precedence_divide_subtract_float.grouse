main {

  print 8.6 / 7.6 - 8.5 nl;
  print 8.2 / 9.7 - 2.7 nl;
  print 3.2 / 2.2 - 4.1 nl;
  print 2.5 / 2.9 - 3.6 nl;
  print 9.5 / 1.8 - 9.4 nl;
  print 9.3 / 1.7 - 2.0 nl;
  print 6.6 / 2.8 - 2.3 nl;
  print 6.6 / 6.4 - 3.8 nl;
  print 9.9 / 7.6 - 4.5 nl;
  print 5.8 / 5.0 - 9.2 nl;
  print 6.1 / 1.9 - 8.9 nl;
  print 9.3 / 6.7 - 6.1 nl;

  print 8.1 - 6.2 / 7.9 nl;
  print 2.0 - 6.8 / 3.6 nl;
  print 5.4 - 8.4 / 3.7 nl;
  print 3.8 - 1.0 / 9.5 nl;
  print 6.7 - 1.5 / 2.0 nl;
  print 8.2 - 6.2 / 9.8 nl;
  print 3.3 - 2.2 / 5.0 nl;
  print 2.8 - 9.8 / 6.2 nl;
  print 5.0 - 6.3 / 9.8 nl;
  print 1.6 - 1.7 / 6.0 nl;
  print 4.9 - 2.8 / 5.5 nl;
  print 6.6 - 6.5 / 2.4 nl;

  print 8.5 / 5.2 - 6.4 / 3.6 nl;
  print 3.9 / 5.6 - 3.6 / 4.4 nl;
  print 6.7 / 8.0 - 9.2 / 2.8 nl;
  print 7.3 / 4.5 - 3.8 / 3.9 nl;
  print 8.4 / 3.1 - 3.5 / 5.7 nl;
  print 3.2 / 9.2 - 4.0 / 4.2 nl;
  print 6.5 / 7.3 - 8.2 / 4.6 nl;
  print 2.0 / 4.9 - 3.3 / 9.9 nl;
  print 2.4 / 6.6 - 4.3 / 2.0 nl;
  print 4.0 / 7.9 - 7.9 / 9.2 nl;
  print 1.0 / 9.8 - 6.2 / 6.9 nl;
  print 1.6 / 1.3 - 7.2 / 2.8 nl;

  print 9.6 - 9.8 / 3.9 - 1.1 nl;
  print 2.0 - 2.8 / 2.8 - 5.6 nl;
  print 7.4 - 2.3 / 1.3 - 4.7 nl;
  print 1.7 - 3.8 / 3.0 - 1.4 nl;
  print 3.1 - 5.2 / 8.0 - 6.5 nl;
  print 4.4 - 7.2 / 1.7 - 4.2 nl;
  print 3.3 - 3.7 / 8.7 - 2.4 nl;
  print 5.0 - 9.6 / 5.6 - 4.1 nl;
  print 9.8 - 1.0 / 2.7 - 2.8 nl;
  print 7.2 - 9.8 / 9.1 - 5.7 nl;
  print 5.0 - 1.7 / 5.1 - 6.8 nl;
  print 3.4 - 8.2 / 7.2 - 8.0 nl;

}

