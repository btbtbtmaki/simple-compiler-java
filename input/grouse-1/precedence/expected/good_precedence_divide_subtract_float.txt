-7.36842
-1.85464
-2.64545
-2.73793
-4.12222
3.47059
0.0571429
-2.76875
-3.19737
-8.04
-5.68947
-4.71194
7.31519
0.111111
3.12973
3.69474
5.95
7.56735
2.86
1.21935
4.35714
1.31667
4.39091
3.89167
-0.143162
-0.121753
-2.44821
0.647863
2.09564
-0.604555
-0.892198
0.0748299
-1.78636
-0.352367
-0.79651
-1.34066
5.98718
-4.6
0.930769
-0.966667
-4.05
-4.03529
0.474713
-0.814286
6.62963
0.423077
-2.13333
-5.73889
